src/PublicApiGeneratorTests/Operator_order.cs

```diff
@@ -0,0 +1,48 @@
+﻿using PublicApiGeneratorTests.Examples;
+using Xunit;
+
+namespace PublicApiGeneratorTests
+{
+    public class Operator_order : ApiGeneratorTestsBase
+    {
+        [Fact]
+        public void Should_show_custom_operators()
+        {
+            AssertPublicApi<ClassWithUnaryOperators>(
+@"namespace PublicApiGeneratorTests.Examples
+{
+    public class ClassWithUnaryOperators
+    {
+        public ClassWithUnaryOperators() { }
+        public static PublicApiGeneratorTests.Examples.ClassWithUnaryOperators op_Addition(PublicApiGeneratorTests.Examples.ClassWithUnaryOperators first, PublicApiGeneratorTests.Examples.ClassWithUnaryOperators second) { }
+        public static PublicApiGeneratorTests.Examples.ClassWithUnaryOperators op_Decrement(PublicApiGeneratorTests.Examples.ClassWithUnaryOperators first) { }
```

Comment:
Would it make sense to have a translation table that translates comnon well known ops into the corresponding operator code? Would be a bit more readable in the output I think

Reply:
I agree. I looked into why the full type name was being generated but, likely due to operators being static, it appears they are seen as "unnested" types which causes the `FullName` to be used instead of just the specific type's name.

Handling the operators in a more special manner would definitely allow me to more cleanly format them (since the problematic area only knows the `Type` without any context of its use). Do you have a preference of such output? To riff on something:

```
public static ClassWithUnaryOperator +(ClassWithUnaryOperator first, ClassWithUnaryOperator second) { }
```

Or are you looking to further condense the signature?